README.md

```diff
@@ -5,7 +5,8 @@
 
 http://coniks.org
 
-##Introduction
+## Introduction
```

Comment:
Looking at some other repos' READMEs, they use "Overview" instead of "Introduction", which I think I like a bit better for this section. What do you think?